src/project/data/Project.hx

```diff
@@ -40,7 +40,7 @@ class Project
 	public function new(path:String)
 	{
 		this.name = "New Project";
-		this.path = Path.resolve(path);
+		this.path = Path.resolve(Path.normalize(path));
```

Comment:
Is this supposed to be `haxe.io.Path`? I don't see an import at the top

Reply:
Nope, thats actually some leftover code i should delete. Its not really doing anything, but why risk it?